app/assets/stylesheets/sbstyles.scss

```diff
@@ -0,0 +1,29 @@
+$primary-orange: #df7401;
+$secondary-orange: #faac58;
+
+body {
+  background-image: image-url("background1.jpg");
+  font-family: "Karma", serif;
+}
+
+h1 {
+  color: $primary-orange;
+  font-weight: bold;
+}
+
+h2 {
+  color: $secondary-orange;
+  font-weight: bold;
+}
+
+.top-bar {
+  margin-bottom: 10px;
+}
+
+.button {
+  margin-right: 10px;
+}
+
+.button a {
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.